ssn/rdf/examples/IBS-TH2-PLUS-brief.ttl

```diff
@@ -0,0 +1,43 @@
+@prefix ex: <https://example.org/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix schema: <http://schema.org/> .
+@prefix sensor: <https://example.org/sensor/> .
+@prefix sosa: <http://www.w3.org/ns/sosa/> .
+@prefix system: <http://www.w3.org/ns/ssn/systems/> .
+@prefix unit: <http://qudt.org/vocab/unit/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+sensor:IBS-TH2-Plus a owl:Class ;
+  rdfs:subClassOf sosa:System ;
+  prov:wasDerivedFrom <https://inkbird.com/products/ibs-th2-plus> ;
+  rdfs:label "IBS-TH2 PLUS Temperature and Humidity Sensor System" ;
+.
+
+sensor:IBS-TH2-Plus-H a owl:Class ;
+  rdfs:subClassOf sosa:Sensor ;
+  rdfs:label "IBS-TH2 PLUS Humidity Sensor type" ;
+.
+
+sensor:IBS-TH2-Plus-T a owl:Class ;
+  rdfs:subClassOf sosa:Sensor ;
+  rdfs:label "IBS-TH2 PLUS Temperature Sensor type" ;
+.
+
+ex:12gth456a-23190 a sensor:IBS-TH2-Plus ; 
+  system:hasSubSystem ex:12gth456a-23190-H ;
+  system:hasSubSystem ex:12gth456a-23190-T ;
+  system:hasSystemCapability sensor:IBS-TH2-Plus-systemCapability ;
+. 
+
+ex:12gth456a-23190-H a sensor:IBS-TH2-Plus-H ; 
+  sosa:observes <http://qudt.org/vocab/quantitykind/RelativeHumidity> ;
+  system:hasSystemCapability sensor:IBS-TH2-Plus-H-systemCapability ;
+. 
+
+ex:12gth456a-23190-T a sensor:IBS-TH2-Plus-T ; 
+  sosa:observes <http://qudt.org/vocab/quantitykind/Temperature> ;
+  system:hasSystemCapability sensor:IBS-TH2-Plus-T-systemCapability ;
+. 
```

Comment:
@dr-shorthair : 

Seeing the conversation about `qk:Temperature` in the other example files, would that make sense to also point to it in this other ttl (for example consistency) ?
On my side I'm ok keeping this example on qudt but I'm less able to investigate the entailment/reasoner aspects that you are.
Example diversity is also good, but some explanation why there are 2 patterns to describe the same observedProperty could help the reader who won't have the discussion between you and @ldesousa at hand when reading the final spec

Reply:
I think I fixed this up to be consistent.